solution/ui/regulations/eregs-component-lib/src/components/CollapseButton.vue

```diff
@@ -68,12 +70,16 @@ export default {
 
     created() {
         this.visible = this.state === "expanded";
-        this.$root.$on("collapse-toggle", this.toggle);
+        eventbus.on("collapse-toggle", this.toggle);
```

Comment:
example of deprecated Vue 2 functionality being replaced by `mitt` eventbus library for Vue 3 compatibility